binr/r2pm/r2pm

```diff
@@ -327,16 +330,24 @@ R2PM_GIT() {
 	cd "${R2PM_GITDIR}"
 	if [ -d "${DIR}" ]; then
 		cd "${DIR}"
-		git checkout master
-		git pull
+		if [ "${R2PM_GITMASTER}" = "master" ]; then
+			git checkout master
+			git pull
+		else
+			git checkout ${R2PM_GITMASTER}
+		fi
 	else
 		git clone --depth 1 --recursive "${URL}" "${DIR}" || R2PM_FAIL "Oops"
 		cd "${DIR}" || R2PM_FAIL "Oops"
 	fi
-	r2 -qv | grep -q git || { # r2 >= 4.5
-		git checkout "r2-${R2_VERSION}"
-		git pull
-	}
+	if [ "${R2PM_GITMASTER}" = "master" ]; then
```

Comment:
this is breaking the feature of building a specific commit when calling it from an r2 release for a package having this variable set